### ISSUE TITLE
Is this project supported?

### ISSUE THREAD

**Issue by viskin** [subitolabs/cordova-gallery-api#1]
Hello,
I have multiple issues on both Android and iPhone.
On android, the library returns same image thumbnail for all returned media.
On iOS, returned thumbnail is always empty string.

**Comment by ebuildy** [comment]
Hello, it's quite an old project, not supported any more. It worked 2 years ago, I don't know nowadays how it looks on modern device / OS as it changes a lot in a few months ;-(

Feel free to fork it if you want, or look at the other forks, maybe there is a more recent code source.

**Comment by ebuildy** [comment]
Try this one : https://github.com/SuryaL/cordova-gallery-api it look quite recent.

**Comment by viskin** [comment]
Cool. Thanks.
I started to implement my own library, but I will take a look at that one.
Mine is here (work-in-progress): https://github.com/terikon/cordova-plugin-photo-library
I want it to work on iOS, and be memory-friendly and fast, so I took approach of getting a Blob object from the plugin, instead of managing files or returning data url. Looks like it works.

**Comment by To1ne** [comment]
I don't believe @SuryaL's fork is really stable. In the current state, the code [does not](https://github.com/SuryaL/cordova-gallery-api/blob/7f289896f903296babc8e5bbf20a9332ac99727b/src/ios/GalleryAPI.m#L140) seems to fill the `data` of images.

**Comment by To1ne** [comment]
Sorry, I'll have to correct myself. @SuryaL's fork is working perfectly, but you need to call `getMediaThumbnail` and `getHQImageData` to actually get the image data.

**Comment by kokkuen** [comment]
Hi,

I'm using @SuryaL's folk but facing the following error.

index.js:7 Uncaught ReferenceError: galleryAPI is not defined(…)(anonymous function) @ index.js:7Channel.fire @ cordova.js:777(anonymous function) @ cordova.js:231

Here are my step :
1. cordova create hello com.example.hello HelloWorld
2. cd hello
3. cordova platform add android --save
4. cordova requirements
5. cordova plugin add https://github.com/subitolabs/cordova-gallery-api.git

use config.xml and www under demo folder to run.

Anyone can help?
Thank you very much.
